tls/s2n_config.c

```diff
@@ -14,6 +14,7 @@
  */
 
 #include <strings.h>
+#include <time.h>
```

Comment:
Do we need this import?

Reply:
I just boosted it from below. I think it's used for the `struct timespec`.